pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -280,6 +429,10 @@ func (l *ListOptionIndexer) notifyEvent(eventType watch.EventType, oldObj any, o
 		}
 	}
 	l.watchersLock.RUnlock()
+
+	l.latestRVLock.Lock()
+	l.latestRV = latestRV
+	defer l.latestRVLock.Unlock()
```

Comment:
Usual style is to put the `defer... .Unlock()` immediately after the `...Lock()` ?

Reply:
Good catch, fixed!